modules/rostests/apitests/shell32/CommandLineToArgvW.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Test for CommandLineToArgvW
+ * COPYRIGHT:   Copyright 2023 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#include "shelltest.h"
+#include <atlstr.h>
+
+static VOID
+DoEntry(INT lineno, LPCWSTR cmdline, INT argc_minus_1, ...)
+{
+    va_list va;
+    va_start(va, argc_minus_1);
+
+    INT real_argc;
+    LPWSTR* real_argv = CommandLineToArgvW(cmdline, &real_argc);
+    INT real_argc_minus_1 = real_argc - 1;
+
+    INT m = min(argc_minus_1, real_argc_minus_1);
```

Comment:
`real_argc_minus_1` is always smaller here?

Reply:
Nope. We don't assume it.

Reply:
Well, line 19 begs to differ I guess.

Reply:
Improved.